File: scholar-web/src/main/webapp/app/model/administration/user/UserSearch.js
Ext.define('scholar.model.administration.user.UserSearch', {
	extend : 'Ext.data.Model',
	fields : [  {
		name : 'id',
		type : 'int'
	},{
		name : 'userName',
		type : 'string'
	}, {
		name : 'userRole',
		type : 'string'
	}, {
		name : 'lastChange',
		type : 'date',
		dateFormat : 'n/j h:ia'
	} ]

});

File: scholar-business/src/main/java/com/varun/scholar/business/data/entities/administration/users/UserSearch.java
package com.varun.scholar.business.data.entities.administration.users;

import java.util.Date;

import javax.persistence.Entity;
import javax.persistence.Temporal;
import javax.persistence.TemporalType;

import com.varun.scholar.business.data.entities.AbstractEntity;

@Entity
public class UserSearch extends AbstractEntity {
	String userName;
	String userRole;
	@Temporal(TemporalType.DATE)
	Date lastChange;
}
